src/main/java/com/movie_collection/bll/helpers/ViewType.java

```diff
@@ -27,6 +27,10 @@ public String getFXMLView() {
         public String getFXMLView() {
             return "views/category-add-edit-view.fxml";
         }
+    }
+    , MEDIA_PLAYER_SELECTION {
+        @Override
+        public String getFXMLView() { return "views/media-player-select-view.fxml"; }
```

Comment:
yeah maybe for such a small window with just one need String its not needed to have a full view ? like leave it but it could also be done by just creating quick stage :)